build/cggnn/Dockerfile

```diff
@@ -1,18 +1,28 @@
-FROM pytorch/pytorch:1.13.0-cuda11.6-cudnn8-runtime
+FROM pytorch/pytorch:2.0.1-cuda11.7-cudnn8-runtime
```

Comment:
I've updated `cg-gnn` and in the original repo it targets cuda 11.8, but there doesn't appear to be a Docker image for this version yet.